### DISCUSSION TITLE
v1.6.0 The "Super Saiyan 5" Performance Update!

### DISCUSSION THREAD

**Discussion by MultimediaLucario** [MultimediaLucario/Lenovo-ThinkPad-T480#12] [Announcements]
![repository-open-graph-template-04 04 2024](https://github.com/MultimediaLucario/Lenovo-ThinkPad-T480/assets/72415505/e82a48c4-c57e-490f-ad61-88f8255167d2)

# The SSJ5 Performance Update!  💻💪

Built on top of the previous release, this update includes a significant change in the ```bootargs``` section in the NVRAM, which is once again thanks to the help and advice from the legend, **[VinylNerd](https://github.com/VinylNerd)**. This change increases the performance of your ThinkPad T480 to what I coined as the "Super Saiyan 5" level of performance, meaning you gain faster speeds, longer battery life **(Varies on the age of your battery)**, and better thermals. All of the very self-explanatory EFI folders include this patch. So, go check it out! 😎💪🔥

<hr /><em>This discussion was created from the release <a href='https://github.com/MultimediaLucario/Lenovo-ThinkPad-T480/releases/tag/v1.6.0'>v1.6.0 The "Super Saiyan 5" Performance Update!</a>.</em>

**Comment by VinylNerd** [comment]
I have some more details to add to this.

the settings you want depend if you want to use HDMI output + Display port out on USB-C 

basically, there is 2 framebuffer options, and the T480 also has a BIOS option to prioritize USB-C or HDMI for the second output, this output is routed to the non thunderbold USB-C.

The entry that will change this is
frambuffer-con2-alldata 02040A00 00080000 87010000 (for HDMI)
frambuffer-con2-alldata 02040A00 00040000 87010000 (for display port)

you will see i have a set of entries for all framebuffer con0,1, and 2 here.

The configuration i have posted below should be the exact same as the default settings if none are entered, but i believe the change from 8 to 4 is needed for dual display port outputs. keep in mind HDMI over USB C Hubs is almost always using DP for transport and converting outside of the computer seeing it.

any questions let me know

![Screenshot 2024-04-05 at 6 00 36 PM](https://github.com/MultimediaLucario/Lenovo-ThinkPad-T480/assets/72950020/82e48518-e482-4cb1-ba43-51b3eda6b7b4)

**Reply by CaBuffalo** [reply]
Where can I fond this setting ? Thanks.

**Comment by VinylNerd** [comment]
https://www.ebay.ca/itm/123394350975

https://www.ebay.ca/itm/281513533371

to get working airdrop, bluetooth and wifi upto Ventura, or on Sonoma using OCLP the combo above will work, you will need to cut the adapter slightly to fit in but its totaly safe as its just a blank pcb, fitting the card in is a squeeze but it works, if you use a bigger card it will not work or close properly.

**Reply by MultimediaLucario** [reply]
Doesn't the ThinkPad T480 have a wifi card whitelist?

**Reply by VinylNerd** [reply]
No only WWAN whitelist, not WLAN. Any card will work

**Comment by VinylNerd** [comment]
https://browser.geekbench.com/v6/cpu/5656212

https://browser.geekbench.com/v6/compute/2031872

https://browser.geekbench.com/v6/compute/2031892

8250u

I have no idea how but somehow i got 5800 on metal and 4200 OpenCL one day about 2 weeks back, it wasn't a fluke because i tested very high CPU, then highest GPU i ever tested on metal and openCL, which tells me there is more tweaking and changes that can be made to slightly improve perf

**Comment by CaBuffalo** [comment]
First of all, thanks for the EFI and great guide. I have installed Sonoma on T480 successfully. But I found that T480 won't go to sleep mode after I close the lid. Is there a fix for this?

Please check.

Thanks.
